sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.institutions.controller.createApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -275,6 +277,7 @@ describe("ApplicationOfferingChangeRequestInstitutionsController(e2e)-createAppl
     });
     application.data.howWillYouBeAttendingTheProgram =
       OfferingIntensity.fullTime;
+    application.offeringIntensity = OfferingIntensity.fullTime;
```

Comment:
Is there a reason not to provide the offering intensity alongside the fatories method like `saveFakeApplicationDisbursements`?
If nor, please allow it to be set alongside them.